conf/timeseries-filodb-server.conf

```diff
@@ -5,6 +5,10 @@ filodb {
     port = 9042
     partition-list-num-groups = 1
   }
+  dataset-configs = [
```

Comment:
Add comments that this defines each dataset and ingestion source

Reply:
Done